packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -9,29 +9,44 @@ export interface LabelVariables {
   startPaddingLeft: string
   endPaddingRight: string
   height: string
-  iconColor: string
+}
+
+export const invertColorScheme = colorScheme => {
+  return Object.keys(colorScheme).reduce((accumulator, key) => {
+    const foreground = colorScheme[key].foreground
+    const background = colorScheme[key].background
+    return {
+      ...accumulator,
+      [key]: {
+        ...colorScheme[key],
+        foreground: background,
+        background: foreground,
+      },
+    }
+  }, {})
 }
 
 export default (siteVars: SiteVariablesPrepared): LabelVariables => {
-  const color = 'rgba(0, 0, 0, 0.6)'
+  const invertedColorScheme = invertColorScheme(siteVars.colorScheme)
```

Comment:
So, we are using `invertColorScheme` only to flip `background` and `foreground`? I am little confused with it because when I see `inverted` I expect this: https://semantic-ui.com/elements/button.html#inverted

The another problem that I will see `foreground` color in `styles`, will try to find it in `colorScheme` and will fail 😕

Reply:
This inverted color scheme is only for the Label component, not in general, and the Label implements the inverted only by flippin the foreground and background - this are the only colors used. The label does not support inverted prop, it's just that it inverts the default color scheme. For the foreground, I don't understand what you mean it is not defined. All colors in the color scheme have the foreground and background colors. Can you provide more details on the second part?

More clarification on the inverted is that, design is considering whether we should add inverted color scheme as part of teams, or whether we should add more foreground and background values in the original one. Until that is decided, we just want to remove the confusion of using foreground on the backgroundColor css prop, and vise versa.

Reply:
> For the foreground, I don't understand what you mean it is not defined. All colors in the color scheme have the foreground and background colors. Can you provide more details on the second part?

https://github.com/stardust-ui/react/pull/1353#discussion-diff-285065169R17

I see there `color: colors.foreground,`, but when I will go to `packages/react/src/themes/teams/colors.ts` there will another value for `foreground` because we flipped them, correct?

Reply:
Yes, but the colors are generated from the variable's color scheme, not the site variables.
```
 const colors = getColorScheme(v.colorScheme, p.color)
```

Reply:
Agree, but as I see in the most cases we simply passing values:
```
colorScheme: siteVars.colorScheme,
```

Reply:
@miroslavstastny can we hear your opinion on this one?